ckanext/harvest/queue.py

```diff
@@ -211,8 +211,14 @@ def __init__(self, redis, routing_key):
     def consume(self, queue):
         while True:
             key, body = self.redis.blpop(self.routing_key)
-            self.redis.set(self.persistance_key(body),
-                           str(datetime.datetime.now()))
+            try:
+                self.redis.set(self.persistance_key(body),
+                            str(datetime.datetime.now()))
```

Comment:
It would be good to get the second argument to line up with the first.

Reply:
Think it was, will reformat the code.